app/serializers/rent_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class RentSerializer < ActiveModel::Serializer
+  attributes :id, :from, :to, :book, :user, :created_at, :updated_at
```

Comment:
for serializers, you can use a `belongs_to` for relationships, just like in the model :D.
That way you also use the serializer you've defined for book and user.